src/groupeddatatable/grouping.jl

```diff
@@ -236,17 +236,35 @@ colwise(sum, groupby(dt, :a))
 ```
 
 """
-colwise(f::Function, d::AbstractDataTable) = Any[vcat(f(d[idx])) for idx in 1:size(d, 2)]
-colwise(f::Function, gd::GroupedDataTable) = map(colwise(f), gd)
-colwise(f::Function) = x -> colwise(f, x)
-colwise(f) = x -> colwise(f, x)
+function colwise(f::Function, d::AbstractDataTable)
```

Comment:
BTW, we shouldn't restrict to `Function` since other types are callable (like constructors). Let's just accept any `f`.